docs/source/whats_new.rst

```diff
@@ -10,6 +10,9 @@ For a full list of included Pull Requests and closed Issues, please see the
 Features
 --------
 
+* John Krasting added the ability to use the oceanic view
```

Comment:
This is for the previous release What's New. We generally make this while doing the release. You can add yourself to the contributors list now though.
https://github.com/SciTools/cartopy/blob/master/docs/source/contributors.rst